moveit_simple_msgs/srv/TrajectoryTrainer.srv

```diff
@@ -0,0 +1,19 @@
+uint8 NEW_WP=0				# Create a new waypoint at current location 			[required: waypoint name, optional: trajectory name]
```

Comment:
This is a pretty complex service.  It's not likely that someone will create the backend code that support this service generically.  Given this, should we just open source our implementation?